static/js/place/chart.tsx

```diff
@@ -148,6 +163,14 @@ class Chart extends React.Component<ChartPropType, ChartStateType> {
     ) {
       return null;
     }
+    if (
+      this.props.config.chartType === chartTypeEnum.CHOROPLETH &&
+      (!this.state.choroplethDataGroup ||
+        this.state.choroplethDataGroup.numDataPoints <
+          CHOROPLETH_MIN_DATAPOINTS)
```

Comment:
let's not reuse that for this. this module should have it's own notion of how many points are required to draw the chart (and it should be some percent of places in the choropleth available)